app/dts/bindings/kscan/zmk,kscan-sideband-behaviors.yaml

```
@@ -0,0 +1,29 @@
+# Copyright (c) 2023, The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: |
+  kscan sideband behavior runner. Only basic system behavior should be used,
```

Typo?
```suggestion
  kscan sideband behavior runner. Only basic system behaviors should be used,
```

Also, I'm not sure how better to describe this, but I think it may be unclear what constitutes "basic system behaviors".

---

I might be able to expand on this, but mostly it's things like tap holds, sticky keys, and any behavior that has non-trivial state stored as part of their use, that can't properly be used with sideband behaviors.

---

Tweaked the comment a bit. Thoughts?

---

Seems reasonable to me. Thanks.